documentation/prometheus-mixin/dashboards.libsonnet

```diff
@@ -148,76 +148,102 @@ local template = grafana.template;
           |||
             rate(
               prometheus_remote_storage_samples_in_total{cluster=~"$cluster", instance=~"$instance"}[5m])
-            - 
+            -
               ignoring(remote_name, url) group_right(instance) (rate(prometheus_remote_storage_succeeded_samples_total{cluster=~"$cluster", instance=~"$instance"}[5m]) or rate(prometheus_remote_storage_samples_total{cluster=~"$cluster", instance=~"$instance"}[5m]))
-            - 
+            -
               (rate(prometheus_remote_storage_dropped_samples_total{cluster=~"$cluster", instance=~"$instance"}[5m]) or rate(prometheus_remote_storage_samples_dropped_total{cluster=~"$cluster", instance=~"$instance"}[5m]))
           |||,
           legendFormat='{{cluster}}:{{instance}} {{remote_name}}:{{url}}'
         ));
 
-      local currentShards =
+      local maxSamplesPerSend =
         graphPanel.new(
-          'Current Shards',
+          'Max samples sent per request',
```

Comment:
can you name this `Max samples per send` so that it matches the queue config fields name.